### PR TITLE
See beauty

### LINKED ISSUE #2: Feruz: Create key styles
- [x] key 
- [x] playing 
- [x] kbd 
- [x] sound

### LINKED ISSUE #3: Feruz: Add classes to key Elements
- [x] add class to html to style

### LINKED ISSUE #4: Feruz : Link to style sheet
- [x] link the style file to html file

### LINKED ISSUE #5: Feruz: Apply background image
- [x] download image for background and apply in css

### PR DESCRIPTION
**Pull request by Feruzteame**
closes  #2 
closes #3 
closes #4 
closes #5

hey guys i try to style it !

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     transition: all 0.07s;
+     width: 100px;
+     text-align: center;
+     color: black;
+     background-color: lightblue;
+     text-shadow: 0 0 5px  black;
+     margin-top: 30%;
+ 
+ }
+ .playing{
```
Comment (aleks2407) [inline style.css]: well, yes) here we have a problem, pls review animate keys miestone in the repo of Evan, .playing style is a part of my user story, as far as this is the animated style . Plus in .playing the border color shpuld change to #ffc600, same as the shadow

**Review by aleks2407 — CHANGES_REQUESTED** [review]
style for .playing should be deleted

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <!-- load the testing dependencies -->
   <script src='./dependencies/chai.js'></script>
   <script src='./dependencies/describe-it.js'></script>
-
+  <link rel="stylesheet" href="style.css">
```
Comment (aleks2407) [inline index.html:13]: there is a link on style.css , see the line 8

File: style.css (hunk 1 above)
Comment (Feruzteame) [reply style.css]: but i think see beauty is all about style. so what do you think ? should i have delete?

File: index.html (hunk 2 above)
Comment (Feruzteame) [reply index.html:13]: yes thank you ! i will delete.

File: style.css (hunk 1 above)
Comment (aleks2407) [reply style.css]: https://github.com/HackYourFutureBelgium/drum-kit/issues/17
I'm just following the steps in evan's repo, as agreed. So if you write it , I have nothing to do on this step, pls check the link

Comment (Feruzteame) [reply style.css]: ok no problem i will remove .

**Review by yildiraykoyuncu — APPROVED** [review]

**Review by abelRoland — APPROVED** [review]

**Review by aleks2407 — APPROVED** [review]

**Pull request merged**